addons/mail/static/src/js/discuss.js

```diff
@@ -933,12 +928,14 @@ var Discuss = Widget.extend(ControlPanelMixin, {
     },
     /**
      * @private
+     * @param {MouseEvent} event
      */
-    _onSettingsButtonClicked: function () {
+    _onSettingsChannel: function (event) {
+        var channel_id = $(event.target).data("channel-id");
```

Comment:
channel_id => channelID

Reply:
not channelId instead of channelID?

Reply:
In the existing function _onUnpinChannel it is channelID.

Reply:
Right, missed that, thanks.